scene/gui/color_picker.cpp

```diff
@@ -1986,13 +2213,23 @@ ColorPicker::ColorPicker() {
 	wheel->set_mouse_filter(MOUSE_FILTER_PASS);
 	wheel->connect(SceneStringName(draw), callable_mp(this, &ColorPicker::_hsv_draw).bind(2, wheel));
 
+	// HACK: I cannot draw focus stylebox on the wheel itself, as it's drawing based on shader.
+	wheel_h_focus_display = memnew(Control);
```

Comment:
In one moment I did. But it meant that I had to add logic for navigating between two focus modes for this control, capturing next/prev actions, because wheel_uv would be used to handle focus for both parts of HSV Wheel.

It added more complexity to the class that is already overloaded with a lot of ifs and was not working well always (like it was sometimes looping focus between those two states when holding `ui_focus_prev` action instead of leaving the control and switching to the hex LineEdit. I thought that adding another control for both drawing the focus stylebox and making switching focus more reliable is a better choice.

If it's better to have few more ifs than using this empty control, I can work on it again.

Reply:
#99515 makes the code cleaner.
If your PR happens to be merged first, I'll simplify it and integrate with my changes.

Reply:
I changed the code to not use a dedicated new Control for handling focus of wheel in HSV Wheel shape. Now the `wheel` control can grab focus in this shape, while focus stylebox is drawn on `wheel_uv`. I had whith `wheel_uv` handling everything and working well with `ui_focus_prev` and `ui_focus_next`, but it `ui_up` was glitching and I choose approach that seemed to work always with some tweaks, without introducing a new control.